### ISSUE TITLE
Subscription patch task failing on latest update

### ISSUE THREAD

**Issue by lae** [lae/ansible-role-proxmox#24]
Looks like something changed with the PVE UI to cause this task to fail:
```
TASK [ansible-role-proxmox : Remove subscription check wrapper function in web UI] ***
task path: /home/travis/build/lae/ansible-role-proxmox/tasks/main.yml:90

Friday 23 February 2018  23:03:57 +0000 (0:00:06.231)       0:00:34.309 ******* 
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: PatchError: 5 out of 5 hunks FAILED
fatal: [test03.lxc]: FAILED! => {
    "changed": false
}
MSG:
5 out of 5 hunks FAILED
```
https://travis-ci.org/lae/ansible-role-proxmox/jobs/345467613

**Comment by ikus060** [comment]
Perfect timing, I was about to fill a bug report !

I recommend to set "failed_when: false" when patching, just to avoid fatal error. If the patch is not applied it's doesn't brake anything, it's just annoying.

**Comment by lae** [comment]
While I can't fix this right now, thoughts on a role variable to disable the subscription patch, @ikus060?

**Comment by ikus060** [comment]
Might be a way to get around the problem. I have some free time today, will check if I can fix the patch too.

**Comment by lae** [comment]
For the time being I released v1.4.8 with the role variable since that's a quick fix.

(I didn't want to explicitly add `failed_when` because then it'd be a lot harder to catch it actually being broken during the automated Travis CI runs.)
